boskos/ranch/ranch_test.go

```diff
@@ -228,7 +228,7 @@ func TestAcquireRoundRobin(t *testing.T) {
 		}
 	}
 
-	for idx, _ := range c.Resources {
```

Comment:
Can we also update that script with `-s` to keep these changes consistent with future changes?